configuration/boot_folder.md

```diff
@@ -33,10 +33,10 @@ Contains many configuration parameters for setting up the Pi. See [the `config.t
 
 Some text-based housekeeping information containing the date and git commit ID of the distribution.
 
-### Device Tree Files
+### Device tree files
```

Comment:
There's no reason why device tree needs to be capitalised, other than if you're going to then shorten it to DT. See https://en.wikipedia.org/wiki/Device_tree

Reply:
Device in this case is only capitalised because it is the first word of the heading, which is how the rest of the documentation does it - i.e. first word capitalised, subsequent words not.

Reply:
I tend to use the same capitalisation on the D and the T - see the heading further down the Wikipedia page.

Reply:
We also seem to have devicetree and DeviceTree, according to devicetree.org. I'll set it back to Device Tree per pelwell.